src/window/sliding_window.rs

```diff
@@ -52,7 +52,7 @@ pub(crate) struct SlidingWindow {
 add_pymethods!(
     SlidingWindow,
     parent: WindowConfig,
-    py_args: (length, offset, align_to),
+    signature: (length=Duration::zero(), offset=Duration::zero(), align_to=DateTime::<Utc>::MIN_UTC),
```

Comment:
This is not right, signature should only have default values if the argument is optional, and this is not the case here, so signature should still be `(length, offset, align_to)`